src/layout/layout.rs

```diff
@@ -1719,5 +1789,120 @@ mod tests {
             ]));
             assert_eq!([a.width, b.width, c.width, d.width], [0, 0, 0, 100]);
         }
+
+        #[test]
+        fn flex() {
+            // length should stretch to the very end
+            let [a] = Rect::new(0, 0, 100, 1).split(&Layout::horizontal([Length(50)]));
+            assert_eq!([a.x, a.width], [0, 100]);
```

Comment:
This test forces the reader to have to look up what the default for Flex is rather than just seeing that directly in front of them. There's two ways to fix this:
1. Create one test that checks that when you create a Layout, the default for flex is a particular value. Then always specify the flex for these tests, knowing that the default is properly handled already. This will make the tests less implicit (and they won't fail when we change the default).
2. Duplicate this test with the same results and with the flex specified

(I prefer option 1)

Reply:
Changed it to rstest. I'll add a separate test for the default.